lib/commands/statusCommand.js

```diff
@@ -22,15 +22,21 @@ module.exports = function (localService, logger, messages,
   }
 
   function print(result) {
-    if (_.isEmpty(_.flatten(_.values(_.omit(result, 'unchanged'))))) {
+    var status = result.status;
+    var conflicts = result.conflicts;
+
+    if (_.isEmpty(_.flatten(_.values(_.omit(status, 'unchanged'))))) {
```

Comment:
if there aren't local changes but there are conflicts shouldn't we let the user know?